automol/vmat.py

```diff
@@ -484,16 +484,15 @@ def dummy_coordinate_names(vma: VMatrix) -> tuple[str]:
     return dummy_names
 
 
-def dummy_source_dict(zma, dir_: bool = True):
+def dummy_source_dict(
+    zma: VMatrix, dir_: bool = True
+) -> dict[int, int] | tuple[int, int]:
```

Comment:
This should be `-> dict[int, int | tuple[int, int]]` (always a dictionary, but the **values** can be either `int` or `tuple[int, int]`).